src/main/java/Unit/Marine.java

```diff
@@ -4,15 +4,11 @@
 import Unit.Interface.Healable;
 import Unit.Interface.Terran;
 
-public class Marine extends GroundUnit implements Terran, BasicControl, Healable {
+public class Marine extends TerranGroundUnit implements Terran, BasicControl, Healable {
 
-    final int MAXHP = 40;
 
-    public Marine(String nickName, int x, int y) {
-        super(nickName, x, y);
-        this.unitName = "마린";
-        this.currentHp = MAXHP;
-        this.atkDamage = 5;
+    public Marine(String nickName, String unitName, int maxHp, int currentHp, int atkDamage, String atkTarget, int x, int y) {
+        super(nickName, unitName, maxHp, currentHp,  atkDamage, atkTarget, x, y);
```

Comment:
```suggestion
    private final String UNIT_NAME = "마린";
    private final int MAX_HP = 40;
    private final int ATTACK_DAMAGE = 5;
    private final String ATTACK_TARGET = "both";
    
    public Marine(String nickName, int x, int y) {
        super(nickName, UNIT_NAME, MAX_HP, MAX_HP,  ATTACK_DAMAGE, ATTACK_TARGET, x, y);
```

위에 SVC 리뷰와 동일한 내용입니다.

Reply:
아.. 제가 원한 게 이런 형태였습니다.. 이렇게 필드값에 미리 설정해둬도 될 내용들은 미리 작성해놓고 객체 생성 시에 필요한 요소만 매개변수로 해서 필드값과 매개변수를 조합하면 되는거였는데.. 열심히 해야겠습니다..